mlflow/utils/autologging_utils/__init__.py

```diff
@@ -446,6 +446,9 @@ def autologging_is_disabled(integration_name):
 
     :param integration_name: An autologging integration flavor name.
     """
+    if _AUTOLOGGING_GLOBALLY_DISABLED:
+        return True
+
```

Comment:
@WeichenXu123 
I think the role of this function is to tell whether or not autologging is disabled for **the specified integration**.